tasks/76_deep-object-clone/deep-object-clone.js

```diff
@@ -2,7 +2,12 @@ export const deepClone = (arg) => {
   let copyObject = {};
   let keysArr = Object.keys(arg);
   for (let i = 0; i < keysArr.length; i++) {
-    copyObject[keysArr[i]] = deepClone(arg[keysArr[i]]);
+    if (arg[keysArr[i]] !== null && typeof arg[keysArr[i]] === 'object') {
```

Comment:
it would be a bit better to have `forEach` here as Viktor write before, or maybe `for of` as it is a bit hard to read like this